lib/util/ast.js

```diff
@@ -51,8 +51,8 @@ function findReturnStatement(node) {
         return nodes[i];
       }
       if (nodes[i].type === 'SwitchStatement') {
-        let j = nodes[i].cases.length - 1;
-        for (; j >= 0; j--) {
+        const j = nodes[i].cases.length - 1;
+        if (j >= 0) {
```

Comment:
```suggestion
        if (nodes[i].cases.length > 0) {
```
this seems a bit clearer?